Documentation/Testing/UnitTesting/Running.rst

```diff
@@ -0,0 +1,129 @@
+:navigation-title: Running
+..  include:: /Includes.rst.txt
+..  _testing-unit-run:
+
+==================
+Running Unit tests
+==================
+
+..  _testing-unit-run-install:
+
+Install phpunit and the TYPO3 testing framework
```

Comment:
I think the "official" name here is PHPUnit. Since TYPO3 is so specific with its proper spelling we might want to honor this too 😅

Reply:
Ack